flink-streaming-java/pom.xml

```diff
@@ -108,6 +108,7 @@ under the License.
 				<artifactId>maven-surefire-plugin</artifactId>
 				<configuration>
 					<reuseForks>false</reuseForks>
+					<argLine>-XX:-UseSplitVerifier</argLine>
```

Comment:
This seems somehow wrong to me. Would be good to get the tests working without changing other modules.